src/cmd/flux-job.c

```diff
@@ -2211,13 +2211,13 @@ int cmd_attach (optparse_t *p, int argc, char **argv)
         log_err_exit ("flux_get_reactor");
 
     /*  Check for the event name that attach should wait for in the
-     *   main job eventlog. The default is the "clean" event.
+     *   main job eventlog. The default is the "finish" event.
```

Comment:
```
On systems with long running job or manager epilogs, or where
otherwise there might be a long delay between a job finishing and the
job's clean event, it is probably unexpected to have flux-job attach
and therefore flux-mini run hang until these post job operations
complete.
```

Commit message seems a little run on.  Perhaps

```
Sometimes there might be a long delay between a job finishing and the
job's clean event, such as with a long running manager epilog.
It is probably unexpected to have flux-job attach, and therefore flux-mini run,
hang until these post job operations  complete.
```